components/ChallengeMaterial.tsx

```diff
@@ -172,14 +191,39 @@ export const ChallengeQuestionCard: React.FC<ChallengeQuestionCardProps> = ({
           </div>
           <div className="card-body">
             <div className="rounded-lg overflow-hidden">
-              {
-                <DiffView
-                  diff={diff}
-                  id={currentChallenge.submission!.id}
-                  comments={currentChallenge.submission?.comments}
-                  status={currentChallenge.status}
-                />
-              }
+              <DiffView
+                diff={diff}
+                id={currentChallenge.submission!.id}
+                comments={currentChallenge.submission?.comments}
+                status={currentChallenge.status}
+              />
+              {comments && <SubmissionComments comments={comments} />}
+              {currentChallenge.submission!.status === 'open' && (
```

Comment:
Use the submission status enum from graphql/index.tsx instead of strings.

https://github.com/garageScript/c0d3-app/blob/687fd8b9449cc920f291695021b7a6a21ba51dac/graphql/index.tsx#L278-L282

Reply:
Fixed.